chef/cookbooks/nova/recipes/config.rb

```diff
@@ -329,7 +316,6 @@
             :keystone_settings => keystone_settings,
             :cinder_insecure => cinder_insecure || keystone_settings['insecure'],
             :ceph_user => ceph_user,
```

Comment:
@mjura: can you handle ceph_user and ceph_uuid in the same way? Either totally drop both of them (like you did for the uuid), or keep both variables (and attributes in the schema) but do not write them in nova.conf (like you did for the user).